tpm2tools/flush_active_handles_test.go

```diff
@@ -0,0 +1,100 @@
+package tpm2tools
+
+import (
+    "crypto/rand"
+    "crypto/rsa"
+    "testing"
+
+    "github.com/google/go-tpm/tpm2"
+
+    "github.com/google/go-tpm-tools/simulator"
+)
+
+const (
+    // How many keys/handles can the simulator contain at once.
```

Comment:
This makes it sound like our simulator can only possibly hold 3 values. Make it clear that this is just how many we are trying for our tests.

Reply:
Done.